README.md

```diff
@@ -1,29 +1,38 @@
+<div align="center">
+<img src="https://raw.githubusercontent.com/AMereBagatelle/fabricskyboxes/1.19.x-dev/src/main/resources/assets/fabricskyboxes/fabricskyboxes_logo.png" width="150">
```

Comment:
Doesn't really add much, to be honest.  Doesn't look great in the rich diff either.  If a better logo is made in the future, I will consider adding it to the readme.

Reply:
Ok I'll remove it. Just tell me if you want me to remove the alignment too